improver/metadata/amend.py

```diff
@@ -84,6 +102,7 @@ def amend_attributes(cube: Cube, attributes_dict: Dict[str, Any]) -> None:
             date format, then this string is replaced with the current
             wall-clock time, formatted as specified.
     """
+    attributes_dict = _load_json(attributes_dict)
```

Comment:
```suggestion
    if isinstance(attributes_dict, str):
        attributes_dict = _load_json(attributes_dict)
```

Reply:
OK I'll define the conditional here instead of in `_load_json` 👍